### PR TITLE
Sh add shortcutkey

### PR DESCRIPTION
**Pull request by toh-sie-hung**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

# Tested on?

- [x] Windows  
- [ ] Linux Ubuntu 
- [ ] Centos 
- [ ] Mac  
- [ ] Others  (State here -> xxx )  

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged

### CODE REVIEW

File: shortcut-key-list.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+## Introduction
+
+The interface of Classifai is user-friendly to use and very easy to master it. For make user's task more easy to handle, 
+these are few common and useful shortcut keys have been created. 
+
+## 1.0 Bounding Box
+
+![Bounding-box-keyboard](/metadata/bounding-box-keyboard.png)
+
+| No | Shortcut Key | Explanation |
```
Comment (BobbyLCW) [inline shortcut-key-list.md:11]: Functionality instead of Explanation

File: shortcut-key-list.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+| 3  | ctrl + z     | undo stages |
+| 4  | ctrl + c     | copy a single boundingbox|
+| 5  | ctrl + v     | paste a single boundingbox |
+| 6  | delete/backspace | delete a selected boundingbox |
+| 7  | double click on selected boundingbox (when click drawing tool) | label selection list appear |
+
+## 2.0 Segmentation
+![Segmentation-keyboard](/metadata/segmentation-keyboard.png)
+
+| No | Shortcut Key | Explanation |
```
Comment (BobbyLCW) [inline shortcut-key-list.md:24]: Functionality instead of Explanation

File: shortcut-key-list.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+The interface of Classifai is user-friendly to use and very easy to master it. For make user's task more easy to handle, 
+these are few common and useful shortcut keys have been created. 
+
+## 1.0 Bounding Box
+
+![Bounding-box-keyboard](/metadata/bounding-box-keyboard.png)
+
+| No | Shortcut Key | Explanation | 
+|----| -------------|-------------|
+| 1  | ctrl + alt   | toggle green light on/off (turn on: move single boundingbox by using arrow key) (turn off: left/right key to next/previous images) |
```
Comment (BobbyLCW) [inline shortcut-key-list.md:13]: toggle green light on/off:

-  on: move single boundingbox by using arrow key.

- off: left/right key to change to next/previous images

File: shortcut-key-list.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+![Segmentation-keyboard](/metadata/segmentation-keyboard.png)
+
+| No | Shortcut Key | Explanation |
+|----| -------------|-------------|
+| 1  | ctrl + alt   | toggle green light on/off (turn on: move single polygon by using arrow key) (turn off: left/right key to next/previous images) |
+| 2  | ctrl + shift | toggle between draw Polygon & move image | 
+| 3  | ctrl + z     | undo stages |
+| 4  | ctrl + c     | copy a single polygon |
+| 5  | ctrl + v     | paste a single polygon |
+| 6  | delete/backspace | delete a selected polygon, delete last drawing dot |
```
Comment (BobbyLCW) [inline shortcut-key-list.md:31]: - delete a selected polygon
- delete last drawing dot (during drawing new polygon)

**Review by BobbyLCW — CHANGES_REQUESTED** [review]
Added comment, please make changes accordingly.Thanks

**Comment by toh-sie-hung** [comment]
okay, np

File: shortcut-key-list.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+| ctrl + shift | toggle between draw boundingbox & move image| 
+| ctrl + alt   | toggle green light on/off <ul><li>on: move single boundingbox by using arrow key</li> <li>off: left/right key to change to next/previous images</li></ui> |
+| ctrl + c     | copy a single boundingbox|
+| ctrl + v     | paste a single boundingbox |
+| ctrl + z     | undo stages |
+| delete/backspace | delete a selected boundingbox |
+| double click on selected boundingbox (when click drawing tool) | label selection list appear |
+
+## 2.0 Segmentation
+![Segmentation-keyboard](/source/segmentation-keyboard.png)
```
Comment (BobbyLCW) [inline shortcut-key-list.md:22]: image disappear?

File: shortcut-key-list.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
+# Classifai Keyboard Shortcut 
+## Introduction
+
+The interface of Classifai is user-friendly to use and very easy to master it. For make user's task more easy to handle, 
+these are few common and useful shortcut keys have been created. 
+
+## 1.0 Bounding Box
+
+![Bounding-box-keyboard](/source/bounding-box-keyboard.png)
```
Comment (BobbyLCW) [inline shortcut-key-list.md:9]: image disappear?

**Review by BobbyLCW — CHANGES_REQUESTED** [review]
Please check the image

**Review by BobbyLCW — APPROVED** [review]
Looks good to me

**Review by codenamewei — APPROVED** [review]

**Pull request merged**